package.json

```diff
@@ -24,11 +24,13 @@
     "storybook": "BROWSER=NONE start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "docs": "start-storybook --docs",
-    "build-docs": "build-storybook --docs"
+    "build-docs": "build-storybook --docs",
+    "check-types": "tsc --noEmit"
```

Comment:
maybe we can call it  `build:check`?  for coherence with the other tsc tasks?

Reply:
`build:check` implies a type of build -- `namespace::field`
it's not building anything, it can't build anything because only webpack can with our config.
It's similar to a linter in what it does,but nota linter